docs/csharp/language-reference/language-specification.md

```diff
@@ -34,8 +34,10 @@ translation.priority.ht:
 # C# Language Specification
 The C# Language Specification is the definitive source for C# syntax and usage. This spec contains detailed information about all aspects of the language, including many points that the documentation for Visual C# doesn't cover.  
   
- You can download this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2013, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 12.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2013 don't include this file.  
+You can download version 5.0 of this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2015, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 14.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2015 don't include this file.
```

Comment:
given that we don't recommend people to use Express anymore and instead use the Community edition (see https://www.visualstudio.com/vs/visual-studio-express/), I don't think this information about Express is needed any longer (and Express is not even listed in the editions comparison on the VS site). 

Also, I'm wondering if the spec will always be in 1033 folder. 1033 folder is the locale for en-us. If I install VS in Brazilian Portuguese, would the spec be somewhere else? I can test this on a VM.